lib/order.rb

```diff
@@ -0,0 +1,21 @@
+class Order
+
+  attr_reader :order
+
+  def initialize
+    @order = {}
+  end
+
+  def add item, number, menu
+    line= menu.select_food(item.to_s)
+    line.each{|k,v| k=k, v[1]=number}
+    @order.merge!(line)
+  end
+
+  def total
+    tot_order=[]
+    @order.values.each{|v| tot_order<< v[0]*v[1]}
```

Comment:
This is quite dense. Extract these parts to a couple of private methods so it's far easier to read.